doc/examples/voidlinux-musl

```diff
@@ -0,0 +1,82 @@
+image:
+  distribution: voidlinux
+  description: Void Linux
+  expiry: 30d
+  architecture: x86_64
+
+source:
+  downloader: voidlinux-http
+  url: https://alpha.de.repo.voidlinux.org/live/current
+  keys:
+    - CF24B9C038097D8A44958E2C8DEBDA68B48282A4
+  variant: musl
```

Comment:
Not really a review comment so much as a question for when we start building those images.
How widely is the musl build used? Is it mostly a curiosity or does it have lots of users?
(Wondering if we should publish images for it)

Reply:
Sorry, but I don't know. You'd have to ask the Void Linux team for their download statistics, I suppose. Personally I use it since musl doesn't have as many issues with static linking as glibc does.

Reply:
Void Linux team does not really keep download statistics. There is a popcorn service, but it has only 22 unique installs. Of those 22, 2 are musl.